modules/veteran_verification/app/controllers/veteran_verification/application_controller.rb

```diff
@@ -15,7 +15,7 @@ def authenticate_token
       # Not supported for Client Credentials tokens
       return false if token.blank? || token.client_credentials_token?
 
-      @session = Session.find(token)
+      @session = Session.find(Digest::SHA256.hexdigest(token.to_s))
```

Comment:
This should have similar `@session = Session.find(token) if @session.nil?` fallback logic, right?